### DISCUSSION TITLE
Wrong dependencies to Microsoft.NET.StringTools at NuGet package.

### DISCUSSION THREAD

**Discussion by arcarrillo** [dotnet/msbuild#7912] [General] [answered]
### Problem description
When the cli tries to add Microsoft.Build.Utilities.Core package at project after version 17.3.0, there is a dependency of this package to Microsoft.NET.StringTools with version 17.3.0 too. This version doesn't exist at nuget.org.

It seems that the version label of Microsoft.NET.StringTools package changed at [this commit](https://github.com/dotnet/msbuild/commit/12c802eab259d8bf2638276f0e6c9a2a64ef3863) aligning it with the version of the solution and the other projects, but it is not being published at nuget.org.

### Possible solutions

- Change the deploy pipeline to publish Microsoft.NET.StringTools at nuget.org with version label 17.x.x.
- Revert this commit to previous version. So this should not be the preferable solution.

**Comment by rainersigwald** [comment, marked as answer]
Thanks for the report--we'll get it published manually ASAP!

**Reply by rainersigwald** [reply]
Done yesterday: https://www.nuget.org/packages/Microsoft.NET.StringTools/17.3.1. 17.3.1 and 17.3.0 are identical for this package.

**Reply by arcarrillo** [reply]
I saw today!
Thanks!
